nimble/host/src/ble_hs_conn.c

```diff
@@ -552,7 +554,7 @@ ble_hs_conn_timer(void)
      */
     if (conn_handle != BLE_HS_CONN_HANDLE_NONE) {
         ble_gap_terminate(conn_handle, BLE_ERR_REM_USER_CONN_TERM);
-        return ble_hs_conn_timer();
+        goto again;
```

Comment:
you need to update comment as well
and `while` or `do {} while` would be nicer than `goto`

Reply:
will do!